lambdas/cfn-list-transform-macro/list_transform/app.py

```diff
@@ -0,0 +1,32 @@
+import json
+import traceback
+
+
+def handle_transform(parameters):
+  operation = parameters['Operation']
+  transform_string = parameters['TransformString']
+  fragment = []
+  input_list = parameters['List']
```

Comment:
should this be `parameters['InputList']`?

also maybe you want to check that it's actually a list?

Reply:
No, it should not be `parameters['InputList']` but I will check that it's a list.

Reply:
check and test for check added in b20ab10